doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -351,7 +351,13 @@ Bluetooth Fast Pair samples
 
 * :ref:`fast_pair_locator_tag` sample:
 
-  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+  * Added:
+
+    * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+    * Support for the firmware update intents on the Android platform.
+      Integrated the new connection authentication callback from the FMDN module and the Device Information Service (DIS) to support firmware version read operation over the Firmware Revision characteristic.
```

Comment:
btw. Could you also check if the intent wouldn't work out-of-the box for `input_device` sample (IIRC we support DIS there by default)?

Reply:
I can do it in the tracking ticket during my follow-up work (tests with Android)